openslides/users/static/templates/users/user-detail.html

```diff
@@ -33,7 +33,7 @@ <h2 translate>Participant</h2>
       <label translate>Participant number</label>
         {{ user.number }}
       <label translate>About me</label>
-        <div ng-bind-html="user.about_me"></div>
+        <div ng-bind-html="user.about_me | trusted"></div>
```

Comment:
We use bleach for validate html code, see [utils/validate.py](https://github.com/OpenSlides/OpenSlides/blob/aade91cead8d92db1e64619a6914c5ba4203b68a/openslides/utils/validate.py). Script tags are not possible, e.g..